### PR TITLE
Merge paradox/scala/cluster-client.md and java/cluster-client.md

### PR DESCRIPTION
**Pull request by richardimaoka**
Refs #23052

## Rendered HTML:

- https://richard-akka-doc.firebaseapp.com/scala/cluster-client.html
- https://richard-akka-doc.firebaseapp.com/java/cluster-client.html

## I split this in two commits:

- **1st commit:** Identical contents in scala/cluster-client.md and java/cluster-client.md -> **this is to take the diff both in scala/cluster-client.md and java/cluster-client.md**

  - [scala/cluster-client.md  (prev) vs. scala/cluster-client.md (after)](https://github.com/richard-imaoka/akka/commit/796d71ed78a088ab79ca08cb41b3dbbb5e5dd632#diff-237c7bd37b70a792547d2fbb38794350)
  - [java/cluster-client.md (prev) vs. scala/cluster-client.md (after)](https://github.com/richard-imaoka/akka/commit/796d71ed78a088ab79ca08cb41b3dbbb5e5dd632#diff-fedc48fbbb2c95b5d5efcfa3b2a4a2cb)

- **2nd commit:** Replace java/cluster-client.md with a symlink to scala/cluster-client.md

### CODE REVIEW

File: akka-docs/src/main/paradox/scala/cluster-client.md [hunk 1]
```diff
@@ -88,22 +92,36 @@ akka.extensions = ["akka.cluster.client.ClusterClientReceptionist"]
...
 
-@@snip [ClusterClientSpec.scala]($akka$/akka-cluster-tools/src/multi-jvm/scala/akka/cluster/client/ClusterClientSpec.scala) { #client }
+Scala
+:  @@snip [ClusterClientSpec.scala]($akka$/akka-cluster-tools/src/multi-jvm/scala/akka/cluster/client/ClusterClientSpec.scala) { #client }
+
+Java
+:  @@snip [ClusterClientTest.java]($akka$/akka-cluster-tools/src/test/java/akka/cluster/client/ClusterClientTest.java) { #client }
 
-The `initialContacts` parameter is a `Set[ActorPath]`, which can be created like this:
+The `initialContacts` parameter is a @scala[`Set[ActorPath]`,] @java[`Set<ActorPath>`,] which can be created like this:
```
Comment (ktoso) [inline akka-docs/src/main/paradox/scala/cluster-client.md]: I guess the `,` could just be moved out of the `[]` :)

**Comment by ktoso** [comment]
LGTM AFAICS, thanks a lot :)

**Review by ktoso — APPROVED** [review]

File: akka-docs/src/main/paradox/scala/cluster-client.md (hunk 1 above)
Comment (richard-imaoka) [inline akka-docs/src/main/paradox/scala/cluster-client.md]: I see, fixed this in the new commit, and looked ok on locally generated HTML.

**Review by raboof — APPROVED** [review]
Good one, thanks!

**Pull request merged**